src/content/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/anomaly-detection-applied-intelligence.mdx

```diff
@@ -11,32 +11,53 @@ redirects:
+import anomalydetails from 'images/anomaly-details.png'
 
-Notifications for anomalies can be delivered in Slack, or you can set up a webhook to deliver messages when you need them. These events are available for querying, creating custom dashboards, and alerting. After you set up a anomaly detection configuration (a group of apps you're interested in), you can add this configuration as a source. Then the anomalies will be automatically correlated with other data sources via incident intelligence.
+import anomalyslacknotifications from 'images/anomaly-slack-notifications.png'
 
-To learn where anomalies appear, how you can use them to understand potential issues before they become incidents, and create alerts from them, watch this short video (approx. 4:15 minutes).
+With applied intelligence's anomaly detection, New Relic alerts your team of any unusual behavior. New Relic observes your APM-monitored applications and understands baseline, or normal, performance. If behavior deviates from the baseline, we know right away and alert your team so you can address any errors promptly and efficiently. 
 
-<Video
-  id="Dhuo2GdR_7o"
-  type="youtube"
+<img
+  title="A screenshot of the anomaly dashboard in the New Relic UI"
+  alt="A screenshot of the anomaly dashboard in the New Relic UI"
+  src={anomalydashboard}
 />
+<figcaption>The anomaly dashboard where your team can monitor any unusual behavior in your system.   
+  </figcaption>
 
-## Requirements
+## How we use anomalies [#anomalies-in-use]
 
-To use anomaly detection, ensure you have:
+At New Relic, we have our own developers who monitor the health of our own application. Our developers want to know if there are any outliers in our system's performance as soon as possible because we want to make sure that our customers are always able to access the data they need. New Relic's anomaly detection monitors three key golden signals: throughput, error rate and latency. Since we monitor our own application, New Relic developers always understand our baseline performance for these metrics. 
 
-* An APM agent installed for at least one application.
-* To receive notifications in Slack, you'll need to ask your IT administrator to install the New Relic application in [your Slack workspace](https://slack.com/help/articles/202035138-Add-an-app-to-your-workspace).
+So, let's say that one afternoon there's a spike in response time and it's taking longer than usual for our customers to access the New Relic application. New Relic will flag this anomalous behavior because our latency metric data has deviated from its baseline. This doesn't necessarily mean there is a problem, it just indicates that New Relic has registered something out of the ordinary in our system and we should take a deeper look.
```

Comment:
I did not understand this until I read your doc! (or what golden metrics were hah!)